### ISSUE TITLE
I am a researcher and want to be able to search the JCT using the journal abbreviation rather than having to type in the full time in order to make my search quicker

### ISSUE THREAD

**Issue by SallyR2020** [antleaf/jct-user-stories#27]

**Comment by markmacgillivray** [comment]
@paulwalk @richard-jones there is no definitive list of journal abbreviations, and we do not have abbreviations for every journal in the system - crossref does not have abbreviations for every journal in their system either. There is an algorithm for working out the short name of a journal, however it cannot be reliably used for all cases because it is not always used when journals create their short names. There are various sources where we could pull from, such as webofknowledge, and we could get a fairly comprehensive list, but we could not guarantee full coverage (some journals do not have short names either, so there is now way to know for sure if we have full coverage or not without also checking all missing ones). If we can define a list of journals that are relevant to JCT, e.g. only those published by the approved list of publishers that JCT provide, then it may be possible to work out if we can get full coverage, or how close we can get.

Either way, there is extra work to do in order to support this, possibly beyond current scope as searching by abbreviation was not originally specified. Another point to consider is whether or not it is really worth it - this use case describes the purpose of short name searching being to avoid typing in the full name, but if the UI utilises autocompletion, then it would not be necessary to type in a full name anyway, just enough characters to differentiate a list down to relevant ones that the user can click on.

**Comment by markmacgillivray** [comment]
As an example, of the ~75k journal names in the large dataset (which is more journals than we'll actually cover), ~4k have ISO 4 abbreviations.

**Comment by paulwalk** [comment]
This is not in the current specification, and will require a certain amount of figuring out and development that has not been factored into he plan, so moving this into the "Candidates for future development" column.

**Comment by hjh33** [comment]
I agree, it doesn't feel like it offers sufficient benefit to users based on the workload required to merit inclusion now.
